cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -397,7 +397,7 @@ def _add_fleet_and_scheduler_resources(self, cleanup_lambda, cleanup_lambda_role
                 slurm_construct=self.scheduler_resources,
             )
 
-    def _add_login_nodes_resources(self):
+    def _add_login_nodes_resources(self, head_eni):
```

Comment:
`_head_eni` is already an instance variable, I don't think we need to pass it as an argument. We can go ahead and use `self._head_eni`.

Reply:
Good catch!

I'll fix it